_posts/2023-03-29-kubeflow-1.7-release.md

```diff
@@ -0,0 +1,190 @@
+* Katib UI: Add authorization mechanisms ([#1983](https://github.com/kubeflow/katib/pull/1983))
+
+<br />
+
+### Training operator
+
+Kubeflow’s unified distributed training operator enhancements include configuration options for fine tuned resource scaling (processor, memory, storage). It now includes HPA support for Pytorch Elastic workloads where users can specify target metric/utilization in Job Spec. This is used for automatic scale up/down of Pytorch Job matching demand while ensuring the elastic policy configured by the user.  This simplifies user workflows significantly and reduces operational toil and costs. It supports multiple scheduler types: Kubernetes, volcano, custom.   Major 1.7 training operator features include:
+
+<br />
+
+* PodGroup enhancements([#1574](https://github.com/kubeflow/training-operator/issues/1574))
+*  Integration with other training frameworks - Paddlepaddle([#520](https://github.com/kubeflow/community/issues/520))
+*  Enhancements on Pytorch Elastic training ([#1645](https://github.com/kubeflow/training-operator/issues/1645), [#1626](https://github.com/kubeflow/training-operator/issues/1626))
+*  Support coscheduling plugin ([#1722](https://github.com/kubeflow/training-operator/issues/1722))
+*  [SDK] Create Unify Training Client([#1719](https://github.com/kubeflow/training-operator/pull/1719))
+
+<br />
+
+### Pipelines
+
+Model developers recognize the time-saving pythonic workflows in Kubeflow Pipelines, which speed iteration by not requiring the generation of new images for pre-prod experimentation.   The new V2 UI and SDK in Kubeflow 1.7 provide valuable details on each pipeline step.   This simplifies the correlation and analysis of parameters, metadata and artifacts during iteration.
+
+In 1.7, KFP also includes enhancements for [control flow functionality](https://www.kubeflow.org/docs/components/pipelines/v2/author-a-pipeline/pipelines/#dsl-control-flow-features).  There are multiple scenarios when a model developer needs to use control flow and pipeline topology enhancements to define a sub-graph in a pipeline ([#8326](https://github.com/kubeflow/pipelines/pull/8326)):
+
+<br />
+
+* ParallelFor ( [#8146](https://github.com/kubeflow/pipelines/pull/8146)) : 
+    * Parallelfor allows users to slice large data into smaller batches, which delivers parallel task execution when iterating over elements in a list.
+* Condition([#8179](https://github.com/kubeflow/pipelines/pull/8179), [#8204](https://github.com/kubeflow/pipelines/pull/8204), [#8209](https://github.com/kubeflow/pipelines/pull/8209)): 
+    * Only executed when a certain condition is met (like if-else condition).
+* ExitHandler( [#8220](https://github.com/kubeflow/pipelines/pull/8220)): 
+    * Acts as the last code logic before finishing a pipeline.
+    * The Exit Handler is executed given the pipeline run is successful or failed.
```

Comment:
Hey @chensun I'm more than happy to remove these points if they are inaccurate. On the other hand, I'm not sure if I will be able to showcase best what new features are available in pipelines. As I understand we agreed with @gkcalat that your team would rewrite the pipelines section. Could you please move forward with that?

Reply:
For pipeline, we can highlight the following two features

```markdown
* Pipelines as components: Pipelines can themselves be used as components in other pipelines, just as you would use any other single-step component in a pipeline. ([#8179](https://github.com/kubeflow/pipelines/pull/8179), [#8204](https://github.com/kubeflow/pipelines/pull/8204), [#8209](https://github.com/kubeflow/pipelines/pull/8209), [#8220](https://github.com/kubeflow/pipelines/pull/8220))
* Sub-DAG visualization that allows pipeline users to dive deep into sub-graph components of their pipeline. ([#8326](https://github.com/kubeflow/pipelines/pull/8326))
```

Optionally, we can add a third bullet point as
```markdown
* Miscellaneous bug and vulnerability fixes.
```

Can we make the change in this blogpost PR now? Also made similar changes to the slides.

cc @gkcalat @zijianjoy in case they may have other thoughts.

Reply:
I agree with Chen's approach here.